LEAF_Nexus/js/employeeSelector.js

```diff
@@ -157,24 +157,24 @@ employeeSelector.prototype.search = function() {
 		            	t.selection = '';
```

Comment:
Error conditions? This has success, it should have error as well. I think we have been using console log to output a message stating there was an error. I would ask around for that info.

Reply:
Added the error condition.